operations/mimir-tests/test-autoscaling-generated.yaml

```diff
@@ -776,15 +404,16 @@ spec:
         - -distributor.health-check-ingesters=true
         - -distributor.ingestion-burst-size=200000
         - -distributor.ingestion-rate-limit=10000
-        - -distributor.ring.consul.hostname=consul.default.svc.cluster.local:8500
         - -distributor.ring.prefix=
-        - -distributor.ring.store=consul
-        - -ingester.ring.consul.hostname=consul.default.svc.cluster.local:8500
+        - -distributor.ring.store=memberlist
         - -ingester.ring.heartbeat-timeout=10m
         - -ingester.ring.prefix=
         - -ingester.ring.replication-factor=3
-        - -ingester.ring.store=consul
+        - -ingester.ring.store=memberlist
         - -mem-ballast-size-bytes=1073741824
+        - -memberlist.abort-if-join-fails=false
```

Comment:
Not in this PR: should we change the default to false, given we recommend to disable it?

Reply:
👍 Let's do that.

Reply:
https://github.com/grafana/dskit/pull/176